openreview/venue/venue.py

```diff
@@ -778,7 +778,7 @@ def send_notification(note):
                 message = messages[decision_note['content']['decision']['value']]
                 final_message = message.replace("{{submission_title}}", note.content['title']['value'])
                 final_message = final_message.replace("{{forum_url}}", f'https://openreview.net/forum?id={note.id}')
-                self.client.post_message(subject, recipients=note.content['authorids']['value'], message=final_message)
+                self.client.post_message(subject, recipients=note.content['authorids']['value'], message=final_message, parentGroup=self.get_authors_id())
```

Comment:
Does this work?

I think the parentGroup checks the recipient is a direct member of the group not a transitive member. We can use the submission author group instead.